test_runner/src/main/kotlin/ftl/filter/TestFilters.kt

```diff
@@ -150,12 +150,20 @@ object TestFilters {
         }
     )
 
-    private fun withClassName(classNames: List<String>): TestFilter = TestFilter(
-        describe = "withClassName (${classNames.joinToString(", ")})",
-        shouldRun = { testMethod ->
-            withPackageName(classNames).shouldRun(testMethod)
-        }
-    )
+    private fun withClassName(classNames: List<String>): TestFilter {
+        val splittedClassNames = classNames.map { it.split("#") }
```

Comment:
it.split make and class + method name, do you mean 
`val splittedClassNames = classNames.map { it.split("#").first() }` ?

Reply:
I need to preserve information about the array size of the splitted string.